src/components/Button/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styles from "./styles/styles.module.scss";
+
+const Button = ({ title, path }) => {
```

Comment:
If we consider this as a generic component, we should define the props more generic. Meaning:
1. A button has `text`. Not `title`.
2. A button has different types like `primary`, `success`, `info`, etc (define whatever that is necessary in your case). This would impact on styling and behavior.

Reply:
Thankssssss Done